core/shared/src/main/scala/org/http4s/multipart/Part.scala

```diff
@@ -44,16 +42,13 @@ object Part {
   def formData(name: String, value: String, headers: Header.ToRaw*): Part[fs2.Pure] =
     Part(
       Headers(`Content-Disposition`("form-data", Map(ci"name" -> name))).put(headers: _*),
-      Entity.Strict(ByteVector.view(value.getBytes(UTF_8))),
+      Entity.string(value),
```

Comment:
Previously charset was set explicitly. Shouldn't we do the same? Though it's hard to imagine changing the default charset (and the function's argument default value consequently).

Reply:
Done.